packages/components/src/components/icon/icons.js

```diff
@@ -666,6 +659,14 @@ import './option/ic-option-raise-fall.svg';
 import './option/ic-option-stayin-goesout.svg';
 import './option/ic-option-touch-notouch.svg';
 import './option/ic-option-up-down-asian.svg';
+import './rebranding/ic-rebranding-binary-bot.svg';
```

Comment:
Is there a reason why we rename the icons to `rebranding` instead of just replacing the icons? The word `rebranding` represents a specific action in time which won't make sense anymore in the future.

Reply:
Since we just changed icons in some specific parts of the project, I did it this way so that we can differentiate between the old and new icons.

Reply:
So, the old icons still present on other parts of the system?